### DISCUSSION TITLE
iotcplatform.com

### DISCUSSION THREAD

**Discussion by tfcarlin** [mrlt8/docker-wyze-bridge#713] [General]
I spent quite a few hours trying to figure out why my wyze-bridge install "suddenly" stopped working. Well, it wasn't sudden one bit but rather happened when I blacklisted iotcplatform.com on pi-hole.  

@mrlt8 Is there any way to bypass iotcplatform or is this required by the API? 

The wyze apps on iOS and Android worked fine with the blacklist.

Thanks for this awesome solution!

**Comment by mrlt8** [comment]
hmm. Will have to look into this. We're trying to use the same methods as the official app, so it should work the same way as the app. 
I believe we may be using a slightly newer version of the SDK, so that may be the reason?
